index.js

```diff
@@ -50,6 +50,7 @@ module.exports = function (options = {}) {
 		 */
 		resolveId(importee, importer) {
 			if (cache_emit.has(importee)) return importee;
+			if (importee === 'svelte' && compilerOptions.generate === 'ssr') return this.resolve('svelte/ssr', importer, { skipSelf: true });
```

Comment:
need to check if the svelte version supports `svelte/ssr`

Reply:
maybe just test if svelte/ssr resolved and only return it if it did?